Yank/yank.py

```diff
@@ -480,6 +460,33 @@ def _create_phase(self, thermodynamic_state, alchemical_phase, restraint_type):
         alchemical_system = factory.alchemically_modified_system
         thermodynamic_state.system = alchemical_system
 
+        # Create the expanded cutoff decoupled state
+        if fully_interacting_expanded_state is None:
+            noninteracting_expanded_state = None
+        else:
+            # Create the system for noninteracting
+            noninteracting_expanded_system = copy.deepcopy(alchemical_system)
+            # Set all USED alchemical interactions to 0
+            # Determine which parameters were used
+            alchemical_parameters = alchemical_states[0].keys()
+            # Set the default parameter values
+            # We dont know which force has which variable, so try all
+            for force in noninteracting_expanded_system.getForces():
+                try:
+                    number_global = force.getNumGlobalParameters()
+                    for parameter_index in range(number_global):
+                        global_name = force.getGlobalParameterName(parameter_index)
+                        if global_name in alchemical_parameters:
+                            force.setGlobalParameterDefaultValue(parameter_index, 0)
+                except:
+                    pass
```

Comment:
Ah! Sorry, I didn't understand that this was the goal here. We can easily do this by something like
```
noninteracting_expanded_system = copy.deepcopy(alchemical_system)
alchemical_state = alchemical_states[-1]
AbsoluteAlchemicalFactory.perturbSystem(noninteracting_expanded_system, alchemical_state)
```
which does the same thing basically without the need to hardcode setting everything to 0.